src/css/custom.css

```
@@ -481,3 +481,9 @@ span.callout + p::after {
 .hero--primary {
   background-image: url("/static/img/hero-bg.png");
 }
+
+/* Remove badges from sidebar */
+a.table-of-contents__link.toc-highlight > span {
```

```suggestion
a.table-of-contents__link.toc-highlight > span.badge {
```

non-blocking: any concerns about this accidentally hiding `span` elements that aren't badges? I don't know what those would be, but it also doesn't seem harmful to add a little more precision here.